Kickstarter-iOS/Features/MessageBanner/Controller/MessageBannerViewController.swift

```diff
@@ -66,7 +67,13 @@ public final class MessageBannerViewController: UIViewController, NibLoading {
     self.viewModel.outputs.messageBannerViewIsHidden
       .observeForUI()
       .observeValues { [weak self] isHidden in
-        self?.showViewAndAnimate(isHidden)
+        guard let self else { return }
+
+        if isHidden, !self.dissmissable {
+          return
+        }
+
+        self.showViewAndAnimate(isHidden)
```

Comment:
Obviously this isn't part of this PR, but what a confusing name! Something like `self.animate(showView: !isHidden)` would be clearer to me. Ah well, something for a future fix.